ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -360,13 +436,6 @@ module type AST = sig
   val match_jane_syntax : ast -> (Embedded_name.t * ast) option
 end
 
-(* Some extensions written before this file existed are handled in their own
-   way; this function filters them out. *)
-let uniformly_handled_extension name =
-  match name with
-  | "local"|"global"|"nonlocal"|"escape"|"curry" -> false
-  | _ -> true
-
```

Comment:
We can't drop these yet, can we?

Reply:
We already sort of did, I think. When we switched the root from `ocaml` to `jane`, this code stopped doing anything useful. (Correct me if I'm wrong; I'm relatively low-confidence on this.)

Reply:
Ohh, you know what?  You're right!